assemblies/docker/Dockerfile

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 ################################################################################
 
-FROM openjdk:8-jre
+FROM openjdk:11-ea-28-slim
```

Comment:
This is an obsolete, early-access image; please don’t use that.

Reply:
I put version 11 also supported by Karaf. If you consider first there was version 8!

Reply:
The problem is not about the version (11 is fine and good idea) it's more the ea-28-slim which is a early access image. Please consider to use 11-jre (LTS).

Reply:
I’m not objecting to the choice of version, but to the choice of image. The 8-jre images are still maintained, and were last updated 12 days ago, with a recent [build of OpenJDK](https://blog.adoptopenjdk.net/2020/07/adoptopenjdk-8u265-available/); the 11-ea-28-slim images haven’t been touched for two years and have known security issues.

Reply:
Unfortunately the 8-jre alpine version for ARM does not work well on ARM devices because it uses OpenJDK Zero. Of version 11 I only found this image which also works on ARM devices. Since the original dockerfile referred to OpenJDK I did not take into consideration other images such as adoptopenjdk ones.

Reply:
Could this image be used adoptopenjdk: 11-jre-hotspot-bionic? I saw that it also includes ARM architectures and the image is up to date. What are you saying? Could I try this one?

Reply:
`adoptopenjdk:11-jre-hotspot` would be better: they provide Windows builds too, and by referencing `11-jre-hotspot` instead of `...-bionic`, if the Linux-based 11 images switch to a base other than Bionic we’ll still get updates.

Reply:
OK. Then I'll update to `adoptopenjdk:11-jre-hotspot` and do a test on arm (on my Raspberry Pi) to make sure everything works. OK?

Reply:
That would be great, thanks!

Reply:
@skitt and @jbonofre I just installed the `adoptopenjdk: 11-jre-hotspot` based Apache Karaf image for ARM on my Raspberry Pi and it works fine. The image I built is this [amusarra/karaf:4.2.9-dev](https://hub.docker.com/layers/amusarra/karaf/4.2.9-dev/images/sha256-8de5205d81263099628cf7659cdd8c648bd13b12cf9cc036f46d9426992927b5?context=explore)

Let me know if you have any doubts, in case I commit the dockerfile and update the README.

Thanks.

![image](https://user-images.githubusercontent.com/708110/91079519-6f700b80-e644-11ea-86fe-aecb0f031be6.png)